mix.exs

```diff
@@ -56,7 +56,7 @@ defmodule EctoSQL.MixProject do
       {:telemetry, "~> 0.4.0 or ~> 1.0"},
 
       # Drivers
-      {:db_connection, "~> 2.2"},
+      {:db_connection, "~> 2.4 and >= 2.4.1"},
```

Comment:
Perhaps there's a more clever way to do this? I want >= 2.4.1, but I don't want to allow 3.0 :)

Reply:
```suggestion
      {:db_connection, "~> 2.4 and ~> 2.4.1"},
```

Reply:
Did you mean `or`? Because this will allow only 2.4.0 =< x < 2.5.0

Reply:
Ugh, yes, sorry. This instead:

```suggestion
      {:db_connection, "~> 2.5 or ~> 2.4.1"},
```